shared-libs/transitions/src/transitions/mark_for_outbound.js

```diff
@@ -25,6 +27,7 @@ const relevantTo = (doc) => {
     .map(key => [pushes[key], key]);
 };
 
+
 const markForOutbound = (change) => {
```

Comment:
What do you think about adding a unit test for this file? We can test the conditions added with your changes.

Reply:
Yes, I think it's a great idea. I'll be adding a test file for `mark_for_outbound.js`.